### ISSUE TITLE
Script causing high load.

### ISSUE THREAD

**Issue by daMihe** [An0nDev/KeepPCAwake#1]
Coming from superuser, not able to commit there.

Using 0 inside try-block is causing high load (exactly one core is wasted). Please insert a sleep or something useful to wait on.

**Comment by an0ndev** [comment]
Yeah, I'll stick like a half-second sleep in or something, sorry lol
